docs/UserGuide.md

```diff
@@ -72,19 +72,27 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding an employee: `addE`
 
-Adds a person to the address book.
+Adds an employee to the employees list.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `adde n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
```

Comment:
'e' in `adde` not capitalised. Perhaps it should be based on what I see of the other formats you have written below.

Reply:
Good catch, this is also supposed to be `addE`.